examples/marklogic-cloud-connection.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2023 MarkLogic Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* This file provides an example for the users to help them connect to MarkLogic cloud using the apiKey and tenancyUrl
+and write a document in database - example-modules */
+
+const marklogic = require('../');
+
+let db = marklogic.createDatabaseClient({
+    apiKey:     'apiKey', // required
+    host:     'tenancyUrl', // required
```

Comment:
We should still avoid the term "tenancyUrl" , because "url" implies a value like "https://something.marklogic.cloud". We just want the host name here, so something like "https://example.marklogic.cloud" would be good.